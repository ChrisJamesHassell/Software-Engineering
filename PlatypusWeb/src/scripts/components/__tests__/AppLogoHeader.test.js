import React from 'react';
import { shallow, mount, render } from 'enzyme';
import AppLogoHeader from '../Navbar/AppLogoHeader';



describe('AppLogoHeader', () => {

  it('matches the snapshot', () =>{
      const tree = shallow(<AppLogoHeader />)
      expect(tree).toMatchSnapshot()
  })
})