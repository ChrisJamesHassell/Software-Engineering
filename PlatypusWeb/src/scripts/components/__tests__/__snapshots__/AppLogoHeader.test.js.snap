// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLogoHeader matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AppLogoHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <NavbarBrand
          id="logo-brand"
        >
          <img
            alt="white logo"
          />
          <Link
            replace={false}
            to="/"
          >
            <span
              id="brand-platy"
            >
              platy
            </span>
            <span
              id="brand-pus"
            >
              pus
            </span>
          </Link>
        </NavbarBrand>,
        <NavbarToggle />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <img
              alt="white logo"
            />,
            <Link
              replace={false}
              to="/"
            >
              <span
                id="brand-platy"
              >
                platy
              </span>
              <span
                id="brand-pus"
              >
                pus
              </span>
            </Link>,
          ],
          "id": "logo-brand",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "white logo",
              "src": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <span
                  id="brand-platy"
                >
                  platy
                </span>,
                <span
                  id="brand-pus"
                >
                  pus
                </span>,
              ],
              "replace": false,
              "to": "/",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "platy",
                  "id": "brand-platy",
                },
                "ref": null,
                "rendered": "platy",
                "type": "span",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "pus",
                  "id": "brand-pus",
                },
                "ref": null,
                "rendered": "pus",
                "type": "span",
              },
            ],
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <NavbarBrand
            id="logo-brand"
          >
            <img
              alt="white logo"
            />
            <Link
              replace={false}
              to="/"
            >
              <span
                id="brand-platy"
              >
                platy
              </span>
              <span
                id="brand-pus"
              >
                pus
              </span>
            </Link>
          </NavbarBrand>,
          <NavbarToggle />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <img
                alt="white logo"
              />,
              <Link
                replace={false}
                to="/"
              >
                <span
                  id="brand-platy"
                >
                  platy
                </span>
                <span
                  id="brand-pus"
                >
                  pus
                </span>
              </Link>,
            ],
            "id": "logo-brand",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "white logo",
                "src": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <span
                    id="brand-platy"
                  >
                    platy
                  </span>,
                  <span
                    id="brand-pus"
                  >
                    pus
                  </span>,
                ],
                "replace": false,
                "to": "/",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "platy",
                    "id": "brand-platy",
                  },
                  "ref": null,
                  "rendered": "platy",
                  "type": "span",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "pus",
                    "id": "brand-pus",
                  },
                  "ref": null,
                  "rendered": "pus",
                  "type": "span",
                },
              ],
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
